Misc/NEWS

```diff
@@ -70,6 +72,10 @@ Build
 Tests
 -----
 
+- Issue #26657: Fix directory traversal vulnerability with http.server on
```

Comment:
Shouldn't also this be in section "What's New in Python 3.4.7 release candidate 1"?